ts/WoltLabSuite/Core/Ui/Dropdown/Simple.ts

```diff
@@ -222,16 +221,19 @@ function toggle(
 
       notifyCallbacks(containerId, "open");
 
+      let firstListItem: HTMLLIElement | null = null;
       if (!disableAutoFocus) {
         menu.setAttribute("role", "menu");
         menu.tabIndex = -1;
         menu.removeEventListener("keydown", dropdownMenuKeyDown);
         menu.addEventListener("keydown", dropdownMenuKeyDown);
-        menu.querySelectorAll("li").forEach((listItem) => {
-          if (!listItem.clientHeight) return;
-          if (firstListItem === null) firstListItem = listItem;
-          else if (listItem.classList.contains("active")) firstListItem = listItem;
 
+        const nonEmptyListItems = Array.from(menu.querySelectorAll("li")).filter(
+          (listItem) => listItem.clientHeight > 0,
+        );
```

Comment:
```suggestion
        const nonEmptyListItems = Array.from(menu.querySelectorAll("li"))
          .filter((listItem) => listItem.clientHeight > 0);
```

I find this much easier to read, because the `filter()` in the front of the line adds context. Also I'm sort of being used to have chained calls being put into a single line every time.

Reply:
This is what came out of Prettier. I'll see if I can convince it to accept your suggestion, otherwise I'm going to merge.

Reply:
Nope, it does not. After I manually adjust this to your suggestion re-running prettier will move this back to how it was.

Reply:
Okay, I'm fine with that.